.travis.yml

```diff
@@ -12,25 +12,36 @@ before_install:
   - sudo apt-get -y -o Dpkg::Options::="--force-confnew" install docker-ce
   - sudo sudo curl -L https://github.com/docker/compose/releases/download/1.17.0/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose
   - sudo chmod +x /usr/local/bin/docker-compose
-before_script:
-  - npm run docker:build
-script:
-  - npm run docker:test
-deploy:
-  - provider: releases
-    skip_cleanup: true
-    api_key:
-      secure: FWDm5RjvdV39l0Ubmt1rR6It70YyKy69UueV42O90VElrm2D0thx8b3L14dnP7oFz40EYp5Q9AkzGw/dhunyQzjy09nm0Oo4hGlVOLoPeiLSL+U8BdjgUQUcwyqTmn7tZU9hiASz1XbW2PMJ7khgZyPFc65j/tJnXQrnpY7TP50=
-    file:
-      - output/contracts/addresses.json
-      - output/contracts/abi.json
-    on:
-      tags: true
-  - provider: script
-    skip_cleanup: true
-    script: ARTIFACTS=true npm run docker:deploy
-    on:
-      tags: true
+script: if [[ "$TESTS" == "integration" ]]; then
+          npm run docker:test:integration;
+        else
+          npm run docker:test:unit:single -- $TESTS;
+        fi
+jobs:
+  include:
+    - stage: build
+      script: docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD";
+              docker pull augurproject/augur-core:latest;
+              npm run docker:build;
+              npm run docker:push
+      env:
+        - DOCKER_USERNAME=pgebheim
```

Comment:
At some point we should get a team account rather than using your personal account.  Not a priority, but would be good to get rid of the bus factor.  We can use KeyBase for storing raw credentials.

Reply:
Indeed. I have integration accounts for everything else just hadn't done it -- I'll do it now before merge.